### PR TITLE
Regenerate FV test keys and certificates; extend certs to 10 years.

### PR DESCRIPTION
**Pull request by fasaxc**
## Description

The FV test certificates were only valid for a year (oops).  Regenerate them with 10-year lifetimes and include a script to regenerate them again if needed.

Remove the CSRs, which are intermediate files and cannot be reused.

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
TBD
```

## Reminder for the reviewer

Make sure that this PR has the correct labels and milestone set.

Every PR needs one `docs-*` label.

- `docs-pr-required`: This change requires a change to the documentation that has not been completed yet.
- `docs-completed`: This change has all necessary documentation completed.
- `docs-not-required`: This change has no user-facing impact and requires no docs.

Every PR needs one `release-note-*` label.

- `release-note-required`: This PR has user-facing changes. Most PRs should have this label.
- `release-note-not-required`: This PR has no user-facing changes.

Other optional labels:

- `cherry-pick-candidate`: This PR should be cherry-picked to an earlier release. For bug fixes only.
- `needs-operator-pr`: This PR is related to install and requires a corresponding change to the operator.

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Comment by marvin-tigera** [comment]
Removing "merge-when-ready" label due to new commits

**Review by caseydavenport — APPROVED** [review]

**Comment by fasaxc** [comment]
PR hit a flake but I think ti's good to go.

**Pull request merged**
